### DISCUSSION TITLE
Argocd email notifications not triggered

### DISCUSSION THREAD

**Discussion by kaikiat** [argoproj/argo-cd#9655] [Q&A]
Hi everyone ! I am trying to setup my argocd email notifications , however it seems like is not even working ? 

I am referring to the docs over here : https://argocd-notifications.readthedocs.io/en/latest/

I am using Google SMTP relay service. These are the steps that I took.

1) Installing notifications and triggers
```
# Install argocd notifications
kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj-labs/argocd-notifications/release-1.0/manifests/install.yaml
# Install Triggers and Templates from the catalog
kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj-labs/argocd-notifications/release-1.0/catalog/install.yaml
```
2) Creating config map
```
export EMAIL_USER=smtp-server-username \
export PASSWORD=smtp-password
kubectl apply -n argocd -f - << EOF
apiVersion: v1
kind: Secret
metadata:
  name: argocd-notifications-secret
stringData:
  email-username: $EMAIL_USER
  email-password: $PASSWORD
type: Opaque
EOF
```
3) Patching app
```
# Patch configmap & app
kubectl patch app sgdecoding-staging -n argocd -p '{"metadata": {"annotations": {"recipients.argocd-notifications.argoproj.io":"recipient@gmail.com"}}}' --type merge
kubectl patch cm argocd-notifications-cm -n argocd --type merge -p '{"data": {"service.email.gmail": "{ username: $email-username, password: $email-password, host: smtp-relay.gmail.com, port: 465, from: $email-username }" }}'
```

After syncing the application via commits or manually, there are no emails send. I verify it by running the command 
`kubectl logs argocd-notifications-controller-84ccc64f96-xhcgv -n argocd -f | grep 'mail`.

I am wondering if anyone has any insights/solutions to this. Any help will be greatly appreciated, thanks !

**Comment by LunaticGen** [comment]
Did you get any solution on this issue?

**Reply by jaweria925** [reply]
I faced the same issue Kindly share the solution if anyone knows.

**Comment by jaweria925** [comment]
<img width="956" alt="image" src="https://github.com/argoproj/argo-cd/assets/58939782/5724722b-8424-477a-9309-dc3d98e10c1d">

here are the logs
but not getting notifications on gmail.

**Comment by JLO974-2** [comment]
You have to restart the notification server pod
It read the configuration only at startup
And for gmail, you need an application passwordor you get an 5.7.8 or 5.7.9 error
